src/pocketmine/plugin/PluginManager.php

```diff
@@ -258,6 +259,14 @@ public function loadPlugins(string $directory, array $newLoaders = null){
 								continue;
 							}
 
+							if(count($pluginMcpeProtocols = $description->getCompatibleMcpeProtocols()) > 0){
+								$serverMcpeProtocols = [ProtocolInfo::CURRENT_PROTOCOL];
+								if(count(array_intersect($pluginMcpeProtocols, $serverMcpeProtocols)) === 0){
```

Comment:
Why not in_array()? Are you expecting the server to support multiple protocol versions in the future?